_commands/basics/uptime.md

```diff
@@ -0,0 +1,37 @@
+---
+---
+
+uptime
+------
+
+This command shows the current time followed by your computer's uptime.
+
+~~~ bash
+$ uptime
+3:03 up 1 day, 6 hrs, 2 users, load averages: 1.38 1.43 1.42
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+
+#### Load Times
+
+##### The load averages output show usage for the last 1, 5, 15 minute intervals. This helps to diagnose which process is CPU hungry.
+
```

Comment:
Can you make line 20 a bullet point, or just a plain paragraph? Just in the name of consistency across the other documentation. = P